jpyinterpreter/src/main/python/jvm_setup.py

```diff
@@ -2,13 +2,34 @@
 import jpype
 import jpype.imports
 import importlib.resources
-from typing import List
+import os
+from typing import List, ContextManager
+
+
+def _normalize_path(path):
```

Comment:
Let's add a unit test for the new logic.

Reply:
No, I am not unit testing official `importlib` code; https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy

Reply:
It is not about testing the official `importlib` code, but rather adding new logic to our codebase and ensuring it does not break anything now and in the future.

Reply:
Regardless, that's not a blocker to approve it.

Reply:
The entire library will not work if this does not work (this loads the jars); this is also not exposed to the users. This is already tested **just** by unit tests running.